docs/index.rst

```diff
@@ -1,28 +1,28 @@
 
-Elements may be grouped into families (an element may be in more than one family), and requested from the lattice object in those families. Some methods take 'handle' inputs, handles specify whether the request made to the IOC pv is a get (read back-'pytac.RB') or set (set point-'pytac.SP') request.
+Elements may be grouped into families (an element may be in more than one family), and requested from the lattice object in those families. The current control system integrates with EPICS and uses EPICS PV (process variable) objects to tell EPICS which IOC (input/output controller) to communicate with. The type of the PV specifies which operations can be performed, there are two types of PV: readback, which can only be used to retrieve data; and setpoint, which can be used to set a value as well as for retrieving data. A single component may have both types; and so some methods take 'handle' as an argument, this is to tell the control system which PV to use when interfacing with EPICS, readback ('pytac.RB') or setpoint ('pytac.SP').
 
-Data may be requested in ``ENG`` or ``PHYS`` units and will be converted as appropriate.  Two types of unit conversion are available: Polynomial (often used for linear conversion) and Piecewise Cubic Hermite Interpolating Polynomial (Pchip; often used for magnet data where field may not be linear with current).  In the case that measurement data (used to set up the conversion objects) is not in the same units as the physical models, further functions may be given to these objects to complete the conversion correctly.
+Data may be requested or sent in engineering (``ENG``) or physics (``PHYS``) units and will be converted as appropriate. This conversion is a fundamental part of how pytac integrates with the physical accelerator, as physics units are what our description of the accelerator works with (e.g. the current in a magnet) and engineering units are what the IOCs on the physical components use (e.g. the current in a magnet). Two types of unit conversion are available: Polynomial (``PolyUnitConv``; often used for linear conversion) and Piecewise Cubic Hermite Interpolating Polynomial (``PchipUnitConv``; often used for magnet data where field may not be linear with current). In the case that measurement data (used to set up the conversion objects) is not in the same units as the physical models, further functions may be given to these objects to complete the conversion correctly.
```

Comment:
You've put 'e.g. the current in a magnet' for both physics and engineering here. The first one should be 'e.g. the magnetic field inside a magnet'